src/components/WorkOrder/WorkOrderHeader.test.js

```diff
@@ -15,7 +15,7 @@ describe('WorkOrderHeader component', () => {
     description: 'This is an urgent repair description',
     propertyReference: '00014888',
     status: 'Work Complete',
-    priorityCode: 'some code',
+    priorityCode: 3,
```

Comment:
We should probably use the `URGENT_PRIORITY_CODE` const here to save a reader looking up what the number means.